jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -136,14 +137,25 @@ public Builder setEntrypoint(@Nullable List<String> entrypoint) {
       return this;
     }
 
+    /**
+     * Sets the username or UID which the process in the container should run as.
```

Comment:
I believe this can be a mix of user and group.

Reply:
You're right, I changed the javadoc.